django/utils/decorators.py

```diff
@@ -17,13 +17,27 @@ def __get__(self, instance, owner):
         return super(classonlymethod, self).__get__(instance, owner)
 
 
-def method_decorator(decorator):
+def method_decorator(decorator, name=""):
     """
     Converts a function decorator into a method decorator
     """
     # 'func' is a function at the time it is passed to _dec, but will eventually
     # be a method of the class it is defined on.
```

Comment:
It's this part that I think needs to be changed (line 24):

> 'func' is a function at the time it is passed to _dec

There's no longer an argument called `func` that gets passed to `_dec`.